runtime/assemble.sh

```diff
@@ -6,8 +6,8 @@ rm -f bin/*.a
 
 for ext in i imc
 do
-    riscv-none-elf-gcc -c -mabi=ilp32 -march=rv32${ext} -mcmodel=medlow asm.S -o bin/nexus-rt.o
-    riscv-none-elf-ar crs bin/riscv32${ext}-unknown-none-elf.a bin/nexus-rt.o
+    riscv64-elf-gcc -c -mabi=ilp32 -march=rv32${ext} -mcmodel=medlow asm.S -o bin/nexus-rt.o
```

Comment:
I would prefer we either don't change these, or try to detect the OS and swap between them for Linux/MacOS.

Reply:
Good idea. Why didn't I think of it. Updated in c6f7716